plasmapy/formulary/misc.py

```diff
@@ -4,20 +4,16 @@
     "Bohm_diffusion",
     "magnetic_energy_density",
     "magnetic_pressure",
-    "mass_density",
     "thermal_pressure",
 ]
-__aliases__ = ["DB_", "pmag_", "pth_", "rho_", "ub_"]
+__aliases__ = ["DB_", "pmag_", "pth_", "ub_"]
```

Comment:
I am not sure why the final test is failing 🤔 the changelog file in 0.8.1.rst has the module path correct so it should load in.